src/transformers/models/rwkv/modeling_rwkv.py

```diff
@@ -264,7 +264,9 @@ def __init__(self, config, layer_id=0):
             try:
                 load_wkv_cuda_kernel(config.context_length)
             except Exception:
-                logger.info("Could not load the custom CUDA kernel for RWKV attention.")
+                logger.warning_once(
```

Comment:
@amyeroberts Not quite sure about this. This line is in the __init__ of attention layers.

Reply:
In this case, I'd say let's keep it as `info` as it's really just the exception message, so we want it to be unaffected by the cache.

Reply:
OK, will discard this commit